README.md

```diff
@@ -287,6 +287,29 @@ Setting up the rootfs... this may take a bit.
 Successfully built r.j3ss.co/img:latest
 ```
 
+#### Cross Platform
+
+In order to use the `--platform` to cross-build option there are some additional requirements.
+
+[Currently](https://github.com/moby/buildkit/blob/v0.5.0/util/binfmt_misc/detect.go#L14) buildkit supports the following platforms:
```

Comment:
@lutostag It's a little more subtle than that. `--platform` works with `foo/bar` as well, as long as the base image you're building from is `scratch` or includes a version for the `foo/bar` platform. Which means you can use all Dockerfile instructions except `RUN`. The binfmt_misc you're referring to is specifically about executing `RUN` commands. The list here under is the list of platforms that BuildKit will automatically test to see if the platform is supported, but you can also (in BuildKit, I'm not familiar with img) specify `foo/bar` as a supported platform and set up binfmt_misc yourself to handle it correctly.

Reply:
Thanks for the info. Went through and tried to hit the important notes. Hopefully helps to clarify it to future readers.